docs/current_docs/extending/modules/state.mdx

```diff
@@ -44,6 +44,15 @@ Here's an example where one field is exposed as a Dagger Function, while the oth
 ```typescript file=./snippets/state-functions/typescript/index.ts
 ```
 
+</TabItem>
+<TabItem value="php" label="PHP">
+PHP already offers `private`, `protected` and `public` keywords to handle member visibility in a class. However, Dagger will only expose those members of a Dagger module that are explicitly decorated with the `#[DaggerFunction]` decorator. Others will remain private.
```

Comment:
This doesn't seem to behave the same way as in TypeScript. Even though the field is marked as public and decorated with `#[DaggerFunction]`, it is not exposed in `dagger call --help` as shown in the example output.

Ping @charjr for review/advice.